structures/sfu/event.js

```diff
@@ -0,0 +1,20 @@
+const s = require('superstruct')
+const isUuid = require('is-uuid')
+const { zouloudate } = require('../lib')
+const uuid = require('uuid')
+
+const Uuid = s.define('Uuid', (value) => isUuid.v4(value))
+const ZoulouDate = zouloudate(s)
+
+const ENTITY_TYPE = 'sfu/event'
+
+const Event = s.object({
+	id: s.defaulted(s.optional(Uuid), uuid.v4),
+	type: s.defaulted(s.optional(s.literal(ENTITY_TYPE)), ENTITY_TYPE),
+	created_at: s.defaulted(s.optional(ZoulouDate), () => (new Date()).toISOString()),
+	key: s.string(),
```

Comment:
There is a limit for `key` and `source`

key @length(min: 8, max: 128)
source @length(min: 2, max: 64)